### ISSUE TITLE
Cannot install dependencies - npm fails with error

### ISSUE THREAD

**Issue by pashidlos** [visual-knight/platform-community-edition#6]
OS: WIndows 10
node: v10.16.0
npm: 6.9.0
prisma2: 2.0.0-preview016.1

Similar issue in prisma2: https://github.com/prisma/prisma2/issues/1232

npm log
````
PS C:\xxx> npm i

> visual-knight-platform-community-edition@0.0.0 postinstall C:\xxx
> npm run generate:photon


> visual-knight-platform-community-edition@0.0.0 generate:photon C:\xxx
> cd libs/api-interface && prisma2 generate

Error: Problem while parsing the query engine response at C:\xxx\node_modules\prisma2\query-engine-windows.exe. Please wait until the 'prisma query-engine' download completes!
SyntaxError: Unexpected token P in JSON at position 0
    at JSON.parse (<anonymous>)
    at Object.<anonymous> (C:\xxx\node_modules\prisma2\build\index.js:114955:25)
    at Generator.next (<anonymous>)
    at fulfilled (C:\xxx\node_modules\prisma2\build\index.js:114913:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)

**Comment by pashidlos** [comment]
I tried to upgrade to latest version of prisma2 (2.0.0-preview024) and next error I got:

```
Error: Get config {"is_panic":false,"message":"error: Environment variable not found: VK_DATABASE.\n  -->  schema.prisma:6\n   | \n 5 |   provider = \"sqlite\"\r\n 6 |   url      = env(\"VK_DATABASE\")\r\n   | \n\nValidation Error Count: 1","meta":{"full_error":"error: Environment variable not found: VK_DATABASE.\n  -->  schema.prisma:6\n   | \n 5 |   provider = \"sqlite\"\r\n 6 | 
  url      = env(\"VK_DATABASE\")\r\n   | \n\nValidation Error Count: 1"},"error_code":"P1012"}
```

And solution was to move` .env` file with variable next to `schema.prisma`

**Comment by pashidlos** [comment]
I managed to fix that with latest prisma2 version 
here is PR
https://github.com/visual-knight/platform-community-edition/pull/7

**Comment by MitkoTschimev** [comment]
Fixed by #7
